pkg/daemon/daemon.go

```diff
@@ -253,9 +264,28 @@ func (dn *Daemon) addProfileConfig(socketPath string, configFile string, nodePro
 	return nil
 }
 
+// Add fifo scheduling if specified in nodeProfile
+func addScheduling(nodeProfile *ptpv1.PtpProfile, cmdLine string) string {
+	if nodeProfile.PtpSchedulingPolicy != nil && *nodeProfile.PtpSchedulingPolicy == "SCHED_FIFO" {
+		if nodeProfile.PtpSchedulingPriority == nil {
+			glog.Errorf("Priority must be set for SCHED_FIFO; using default scheduling.")
+			return cmdLine
+		}
+		priority := *nodeProfile.PtpSchedulingPriority
```

Comment:
before you do this please check if the variable is not nil.

Reply:
This is done on line 274.

Reply:
Done on line 270